create-dmg

```diff
@@ -4,6 +4,8 @@
 
 # Bail out on any unhandled errors
 set -e;
+# Any command that exits with non-zero code will cause the pipeline to fail
+set -o pipefail;
```

Comment:
Maybe even `-eo`?

Reply:
Thanks for your review.

> Maybe even `-eo`?

Yeah, but IMO, splitting them up makes our code and intention more clear, and new shell developers can also learn something from here.